compiler/linter.nim

```diff
@@ -109,7 +109,8 @@ proc differs(conf: ConfigRef; info: TLineInfo; newName: string): string =
   var first = min(info.col.int, line.len)
   if first < 0: return
   #inc first, skipIgnoreCase(line, "proc ", first)
-  while first > 0 and line[first-1] in Letters: dec first
```

Comment:
The while statement intends to handle stropping variables, right?

Reply:
Yes and iirc also operators.

Reply:
I see